src/BigBlueButton.php

```diff
@@ -116,6 +116,14 @@ class BigBlueButton
      */
     public function __construct(?string $baseUrl = null, ?string $secret = null, ?TransportInterface $transport = null, string $hashingAlgorithm = HashingAlgorithm::SHA_1)
     {
+        if (empty($baseUrl)) {
```

Comment:
```suggestion
        if (null === $baseUrl) {
```
We should use explicit checks here. `empty` on scalars is unpredictable and therefore a PITA ;) .